.buildkite/x-pack/pipeline.xpack.heartbeat.yml

```diff
@@ -166,10 +169,11 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/heartbeat: Windows 2019 Unit Tests"
-  
+
   - group: "x-pack/heartbeat MacOS Extended Tests"
     key: "x-pack-heartbeat-extended-tests-macos"
     if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
+
     steps:
       - label: ":mac: MacOS x86_64 Unit Tests"
```

Comment:
super nit: while you are here and making things consistent, it may be in scope (but totally optional) to replace `MacOS` with `macOS` to make things consistent with the official capitalization of this OS. (should be changed also in `notify.github_commit_status.context`)

Reply:
will update other beats as well